regression/contracts/function_apply_01/main.c

```diff
@@ -0,0 +1,20 @@
+// function_apply_01
+
+// Note that this test is supposed to have an incorrect contract.
+// We verify that applying (without checking) the contract yields success,
+// and that checking the contract yields failure.
+
+#include <assert.h>
+
+int foo() 
+__CPROVER_ensures(__CPROVER_return_value == 0)
```

Comment:
Nit picking: uniformly indent (or don't indent) `__CPROVER_ensures`, `__CPROVER_requires` across this pull request.

Reply:
Addressed in new version.